.github/workflows/ci.yml

```diff
@@ -6,13 +6,16 @@ on:
 
 permissions:
   contents: read
+  actions: write
```

Comment:
Maybe we should put the `actions: write` permission where it's needed on the individual jobs?

I think right now we allow all actions to run with `actions: write` level permissions, I'm not sure if we're not accidentally elevating actions that only need `read` level to higher permissions than they used to have?

> You can use `permissions` either as a top-level key, to apply to all jobs in the workflow, or within specific jobs. When you add the `permissions` key within a specific job, all actions and run commands within that job that use the `GITHUB_TOKEN` gain the access rights you specify.

https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#permissions

Reply:
It will be needed in all jobs in the workflow, that's why I put it as a top-level key.

Reply:
Okay!